src/views/forms/AccountManagementForms/DepositForm.tsx

```diff
@@ -279,6 +279,21 @@ export const DepositForm = ({ onDeposit, onError }: DepositFormProps) => {
 
   const onSubmit = useCallback(
     async (e: FormEvent) => {
+      track(
+        AnalyticsEvents.TransferDepositFundsClick({
```

Comment:
I feel like these should all be ??. Also what happened to the eslint rule that checked for this lol

Reply:
aw man you're totally right! wtf 🤔

Reply:
it's still in our estlinrc, why isn't it being caught by linting 👀